package.json

```diff
@@ -44,17 +44,18 @@
   "dependencies": {
     "atom-linter": "10.0.0",
     "atom-package-deps": "5.0.0",
+    "npm": "^6.7.0",
```

Comment:
🔥 Looks like an extra dependency snuck in here 😉.

Reply:
Ready!!